easybuild/easyconfigs/q/qrupdate/qrupdate-1.1.2-foss-2019a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'qrupdate'
+version = '1.1.2'
+
+homepage = 'https://sourceforge.net/projects/qrupdate/'
+description = """qrupdate is a Fortran library for fast updates of QR and Cholesky decompositions."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
@crubb Move to `GCC` toolchain (not `GCCcore`, since it's Fortran and compiling with `iccifort` is important if this is needed for something built with `intel` toolchain)

Reply:
Moved to `GCC/8.2.20-2.31.1`.